formula-android/src/main/java/com/instacart/formula/FormulaAndroid.kt

```diff
@@ -47,6 +45,22 @@ object FormulaAndroid {
         FormulaFragmentDelegate.fragmentEnvironment = fragmentEnvironment
     }
 
+    /**
+     * Initializes Formula Android integration. Should be called within [Application.onCreate].
```

Comment:
Does it need to be called before the DI injection call in `[Application.onCreate]`? If yes - might be worth mentioning it here.

Reply:
Not necessary because usually, the logic will not run until a fragment is shown.